modules/obtaining-ocp-packages-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,182 @@
+= Obtaining {product-title} packages
+
+Before you can update your Red Hat Enterprise Linux (RHEL) compute machines on a restricted network cluster, you must obtain the required images and components and store them in your repository.
+
+[IMPORTANT]
+====
+You must obtain the required images and software components on a system with the same architecture as the cluster that is in your disconnected environment.
+====
+
+.Prerequisites
+
+* Obtain a Red Hat Enterprise Linux (RHEL) 7 server that you have root access to, with access to the internet and at least 110 GB of disk space. Download the required software repositories and container images to your computer.
+
+* Plan to maintain a webserver within your disconnected environment to serve the mirrored repositories. Copy the repositories from the internet-connected host to this webserver, either over the network or by using physical media in disconnected deployments.
+
+* Provide a source control repository. After installation, your nodes must access source code in a source code repository, such as Git.
++
+When building applications in {product-title}, your build might contain external dependencies, such as a Maven Repository or Gemfiles for Ruby applications.
+
+* Provide a registry within the disconnected environment.
+
+.Procedure
+
+On the RHEL 7 server with an internet connection, sync the repositories:
+
+. To ensure that the packages are not deleted after you sync the repository, import the GPG key:
++
+[source,terminal]
+----
+$ rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+----
+
+. Register the server with the Red Hat Customer Portal. You must use the credentials that are associated with the account that has access to {product-title} subscriptions:
++
+[source,terminal]
+----
+$ subscription-manager register
+----
+
+. Pull the latest subscription data from Red Hat Subscription Manager:
++
+[source,terminal]
+----
+$ subscription-manager refresh
+----
+
+. Attach a subscription that provides {product-title} channels.
+.. Find an available subscription pool that provides {product-title} channels:
++
+[source,terminal]
+----
+$ subscription-manager list --available --matches '*OpenShift*'
+----
+
+.. Attach a pool ID for a subscription that provides {product-title}:
++
+[source,terminal]
+----
+$ subscription-manager attach --pool=<pool_id>
+----
++
+[source,terminal]
+----
+$ subscription-manager repos --disable="*"
+----
+
+. Enable only the repositories required by {product-title} {product-version}.
++
+--
+** For cloud installations and on-premise installations on x86_64 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-4.7-rpms" \
+    --enable="rhel-7-server-ansible-2.9-rpms"
+----
+--
+** For on-premise installations on IBM POWER8 servers, run the following command:
```

Comment:
In https://docs.openshift.com/container-platform/4.6/machine_management/adding-rhel-compute.html#rhel-preparing-playbook-machine_adding-rhel-compute, we did not mentioned non-x86 arch servers, so I think we should remove this part to keep consistent as install.